app/Controllers/authController.php

```diff
@@ -123,8 +123,7 @@ public function formLoginAction() {
 
 			$conf = get_user_configuration($username);
 			if (is_null($conf)) {
-				Minz_Request::bad(_t('feedback.auth.login.invalid'),
-				                  array('c' => 'auth', 'a' => 'login'));
+				Minz_Error::error(403, array(_t('feedback.auth.login.invalid')), false);
```

Comment:
This one was problematic indeed. I have added a `return`.
The other two below should be fine.